README.md

```diff
@@ -91,20 +91,27 @@ Zonemaster comes with localization for these locales:
 
 ## Repositories
 
-The Zonemaster project has four different repositories. All the software for
-the Zonemaster project lies in these four repositories. The software has not yet
-been packaged for any operating systems, and you have to install most of it from
-the source code.
+The Zonemaster Product uses six different Github repositories. The repository that
+this file is found in is the main repository that holds specifications and documentation
+for the Zonemaster product. All the software for the Zonemaster project lies in five 
+repositories, one for each Zonemaster component. The software has not yet been packaged 
+for any operating systems, and you have to install most of it from the source code. The
+recommended method is to install from [CPAN], but it is possible to install directly from 
+clones of the Github repositories.
 
-In order to install and run the different components, have a look at the
-installation documentation under each of the repositories below:
+The Zonemaster Product includes the following components:
 
  * [zonemaster-ldns] - which is a fork of of [ldns] with a Perl frontend.
  * [zonemaster-engine] - which contains the libraries doing the actual tests.
  * [zonemaster-cli] - a Command Line Interface for the engine.
  * [zonemaster-backend] - which interfaces between the GUI and an API with the engine.
- * [zonemaster-gui](https://github.com/dotse/zonemaster-gui) - A graphical user
-   interface for the engine.
+ * [zonemaster-gui] - A graphical user interface for the engine.
+
+## Installation
+
+To install Zonemaster, start with installation of [zonemaster-engine] (which will
+draw in Zonemaster-LDNS) and then continue with the other parts. From the links above
```

Comment:
"From the links above installation instructions you will find." -> "You will find installation instructions from the links above." :)

Reply:
Updated.